types/lil-uri/index.d.ts

```diff
@@ -0,0 +1,207 @@
+// Type definitions for lil-uri 0.2
+// Project: https://github.com/lil-js/uri#readme
+// Definitions by: Wayne Carson <https://github.com/wcarson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Factory function to get a new URI instance.
+ * @param uri
+ */
+export default function uri(uri?: string): URI;
```

Comment:
It seems like the [underlying library](https://www.npmjs.com/package/lil-uri) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

It looks like this is a UMD module so you should follow the template for that.

Reply:
Hi. I've updated the definition to conform to the module-function.d.ts template. Thanks, Wayne